ext/redhat/facter.spec.erb

```diff
@@ -94,11 +94,11 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%{_bindir}/cfacter
+%{_bindir}/facter
 %{_libdir}/libfacter.so*
 %{_includedir}/facter
 %if 0%{?rhel} >= 7 || 0%{?fedora}
-%{vendor_ruby}/cfacter.rb
+%{vendor_ruby}/facter.rb
```

Comment:
I'd probably say yes to that, in which case we really just treat the `--cfacter` option as something that worked with cfacter 0.2.0 to 0.4.0.